modules/programs/yazi.nix

```
@@ -141,7 +141,7 @@ in {
     };
 
     initLua = mkOption {
-      type = with types; nullOr path;
+      type = with types; nullOr (either path str);
```

I think that `lines` is more correct than `str`.

```suggestion
      type = with types; nullOr (either path lines);
```

---

Had to check source for this one, https://github.com/NixOS/nixpkgs/blob/d8724afca4565614164dd81345f6137c4c6eab21/lib/types.nix#L473 it should be good